src/applications/static-pages/health-care-manage-benefits/refill-track-prescriptions-page/components/AuthContent/index.js

```diff
@@ -2,10 +2,29 @@
 import React from 'react';
 // Relative imports.
 import CallToActionWidget from 'platform/site-wide/cta-widget';
+import CernerCallToAction from '../../../components/CernerCallToAction';
+
+const callToActions = [
+  {
+    deriveHeaderText: facilityNames =>
+      `Refill prescriptions from ${facilityNames}`,
+    href: '',
+    label: 'Refill prescriptions on My VA Health',
+  },
+  {
+    deriveHeaderText: () =>
+      `Refill prescriptions from another VA Medical Center`,
+    href: '',
```

Comment:
Need the 2 `href` links [here](https://adhoc.invisionapp.com/share/DYVMI5OXEQN#/screens/404897044):

![image](https://user-images.githubusercontent.com/12773166/86203920-6de20580-bb23-11ea-9701-162b0a786e30.png)

Reply:
From Cerner:

Link to this in Staging My VA Health: https://ehrm-va-test.patientportal.us.healtheintent.com/pages/medications/current
Link to this in PROD My VA Health: https://patientportal.myhealth.va.gov/pages/medications/current